.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: trailing-whitespace
```

Comment:
What the motivation to remove some of the default hooks?

Reply:
Fixed those by adding instructions for interacting with dockerfile on README and descriptive PR body.